pulsar-client/src/main/java/com/yahoo/pulsar/client/api/MessageId.java

```diff
@@ -44,4 +44,11 @@
     public static MessageId fromByteArray(byte[] data) throws IOException {
         return MessageIdImpl.fromByteArray(data);
     }
+    
+    /**
+     * Get messageId in string format
+     * 
+     * @return MessageId in string format
+     */
+    public String getMsgIdAsString(String entitySeparator);
```

Comment:
I think we should change `toString()` method because motivation of this change is when user enables debug log they see long msgId string and everywhere in debug log we just print msgId.toString() so, I think this change will not address actual problem.